client/src/pages/PageNotFound/index.jsx

```diff
@@ -1,47 +1,123 @@
 import React from 'react';
 import { Typography, Button, Box } from '@material-ui/core';
 import { Link } from 'react-router-dom';
-import PropTypes from 'prop-types';
 
-const PageNotFound = ({ location }) => {
-  const { pathname } = location;
+const PageNotFound = () => {
+  const NotFoundImage = () => {
+    return (
+      <Box>
+        <svg
```

Comment:
maybe svg can go in seperate file because it's quite big

Reply:
yes